tokenizer/build.sbt

```diff
@@ -0,0 +1,5 @@
+name := "anorm-tokenizer"
```

Comment:
I think some of the settings (the publish/maven settings) in the main `build.sbt` need to be in here too.  They could be added to the `project/Common.scala` plugin, but that means it will also be added to the docs project, which I'm not sure if we want or not.